packages/runtime/datastore/src/localChannelContext.ts

```diff
@@ -121,6 +121,10 @@ export class LocalChannelContext implements IChannelContext {
         this.services.value.deltaConnection.reSubmit(content, localOpMetadata);
     }
 
+    public rebaseOp() {
```

Comment:
Worth double checking that we do not get here for "attaching" data store. I.e. we did not receive ack for attach message, and have some ops on top of it.

Here is code from dataStores.ts:
```
            if (this.runtime.attachState !== AttachState.Detached) {
                dataStoreContext = new RemotedFluidDataStoreContext(
```
I'm not sure it's correct given addition of this feature. I.e. before this work, we had only two states - either we load from storage (and all data stores are "remote", the are attached) or we load draft (and all data stores are local and have no ops on top of it, they are "detached").
I believe with this feature we are introducing a third option - "attaching" state, In such state, I believe we need to create LocalFluidDataStoreContext when we see attach op in saved local ops, and deliver future ops to such local channel.

Reply:
We absolutely need to cover it with UT. Maybe it's good first step, and ensure that we indeed hit this code, before we address it.